module/os/linux/zfs/zfs_znode.c

```diff
@@ -486,7 +486,7 @@ zfs_set_inode_flags(znode_t *zp, struct inode *ip)
  * updated to access the inode.  This can be done incrementally.
  */
 void
```

Comment:
nit: this is the comment I was suggesting we should update since it's now a bit confusing.  Can you either remove it entirely or remove everything but the first sentence?  Everything else looks good, thanks!

Reply:
I missed that. Done.